node/src/blockchain/blockchain_interface.rs

```diff
@@ -1059,27 +1204,24 @@ mod tests {
         assert_gas_limit_is_not_under(subject, 55000, 65000)
     }
 
-    fn assert_gas_limit_is_not_under<T: Transport + Debug>(
+    fn assert_gas_limit_is_not_under<T: Transport + Debug + 'static>(
```

Comment:
You're passing in a minimum and a maximum here; should the name of this function be something like `assert_gas_limit_is_between`?